I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 243, in script
    w "I'm {b}#%\\!^{/bs}. Ye can just call me {color=#e35460}RIBERTO{/color}."
ValueError: unsupported format character '\' (0x5c) at index 9

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 243, in script
    w "I'm {b}#%\\!^{/bs}. Ye can just call me {color=#e35460}RIBERTO{/color}."
  File "D:\Programas\RenPy\renpy-8.1.1-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "D:\Programas\RenPy\renpy-8.1.1-sdk\renpy\exports.py", line 1462, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character '\' (0x5c) at index 9

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.1.23060707
test01 1.0
Mon Aug 14 01:23:48 2023
